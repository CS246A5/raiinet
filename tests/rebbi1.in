-ability1
-ability2
-ability2 PPFSD

-link1 placement_file3.txt

-graphics
