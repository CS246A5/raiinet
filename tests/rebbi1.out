Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: V1 b: D4 c: V3 d: V2
e: D3 f: V4 g: D2 h: D1
========
abcSSfgh
...de...
........
........
........
........
...DE...
ABCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: ? B: ? C: ? D: ?
E: ? F: ? G: ? H: ?
